lib/ansible/modules/cloud/amazon/ec2_instance.py

```diff
@@ -1515,12 +1568,18 @@ def main():
 
         module.params['filters'] = filters
 
+    if module.params.get('cpu_options'):
+        boto3_version = int(boto3.__version__.replace('.', '')[0:3])
+        if boto3_version < 170:
```

Comment:
please use distutils.version.LooseVersion rather than do this - it will break if boto3 gets to 1.10.0, as 110 is less than 170

Also, this test should be against botocore version, not boto3 version - looks like it was introduced in botocore 1.10.16 (boto3 1.7.0 could be used with 1.10.15, for example, at which point this would fail)

Reply:
Fixed